README.md

```diff
@@ -18,8 +18,8 @@ are:
     dockerhub        Fetch repository data from Docker Hub site
     gerrit           Fetch reviews from a Gerrit server
     git              Fetch commits from Git
-    github           Fetch issues from GitHub
-    gitlab           Fetch issues from GitLab
+    github           Fetch issues, pull requests and repository information from GitHub
+    gitlab           Fetch issues, merge requests and project information from GitLab
```

Comment:
The categories for GitLab are [issues and merge requests](https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/gitlab.py#L87) (project info is not retrieved).

Reply:
Ok, I will update the change.